docs/plugins/event-hierarchy.md

```diff
@@ -7,10 +7,34 @@ slug: /plugins/event-hierarchy
 
 # Event Hierarchy
 
-We have two main events in our `Data Service Generator (DSG)` service. One for the server creation and the other for the admin-ui creation. **These events are running simultaneously, and there is no guarantee which will finish first**
+We have two main event hierarchies in our Data Service Generator (DSG) service: one for Node.js services and one for .NET services. Each of these hierarchies includes events to create the server's files and, in the case of Node.js, the admin-ui files.
 
-Each of these main events includes other events to create the server’s files and the admin-ui files (in our source code it is sometimes referred as and `modules`).
+## Node.js Event Hierarchy
 
-The following diagram represents the execution order of the events in DSG.
+For Node.js services, we have two main events: one for the server creation and the other for the admin-ui creation. **These events are running simultaneously, and there is no guarantee which will finish first.**
 
-![](.\assets\hierarchy.png)
+Each of these main events includes other events to create the server's files and the admin-ui files (in our source code, it is sometimes referred to as `modules`).
+
+For a complete list of Node.js events, descriptions, parameters, and examples, refer to the [Node.js Plugin Events Reference](/plugins/plugin-events/create-server/) section.
+
+The following diagram represents the execution order of the events in DSG for Node.js:
+
+![A Diagram of Node.js Plugins Events](./assets/hierarchy.png)
+
+## .NET Event Hierarchy
+
+For .NET services, the event hierarchy is focused on server-side generation. It includes events for creating various components of the .NET service, such as controllers, services, models, and infrastructure elements.
```

Comment:
What does "models" mean here? 
@mulygottlieb - Does it have a special .NET explanation?

Reply:
Models is probably the DB models. It's the same for .NET and Node